core/src/main/scala/org/apache/spark/scheduler/TaskSetManager.scala

```diff
@@ -36,9 +36,8 @@ import org.apache.spark.util.collection.MedianHeap
 /**
  * Schedules the tasks within a single TaskSet in the TaskSchedulerImpl. This class keeps track of
  * each task, retries tasks if they fail (up to a limited number of times), and
- * handles locality-aware scheduling for this TaskSet via delay scheduling. The main interfaces
- * to it are resourceOffer, which asks the TaskSet whether it wants to run a task on one node,
- * and statusUpdate, which tells it that one of its tasks changed state (e.g. finished).
+ * handles locality-aware scheduling for this TaskSet via delay scheduling. The main interface
+ * to it is resourceOffer, which asks the TaskSet whether it wants to run a task on one node.
```

Comment:
Do you want to mention the `handleSuccessfulTask/handleFailedTask` functions? Actually I don't think this is a big deal since they are still triggered by StatusUpdate.

Reply:
How about replace statusUpdate with handleSuccessfulTask/handleFailedTask? That is more accurate. Something like:

```
 * handles locality-aware scheduling for this TaskSet via delay scheduling. The main interfaces
 * to it are resourceOffer, which asks the TaskSet whether it wants to run a task on one node,
 * and handleSuccessfulTask/handleFailedTask, which tells it that one of its tasks changed state.
```